src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectLaunchTargetsProvider.cs

```diff
@@ -267,6 +268,8 @@ private async Task<bool> IsIntegratedConsoleEnabledAsync()
                 }
             }
 
+            var commandLineArgs = resolvedProfile.CommandLineArgs != null ? Regex.Replace(resolvedProfile.CommandLineArgs, "[\r\n]+", " ") : null;
```

Comment:
You can simplify this to:

``` csharp
string? commandLineArgs = resolvedProfile?.Replace("\r\n", " ").Replace("\r", " ").Replace("\n", " ");
```

Which avoids the need for the explicit null check and the use of a regex. A regex is a fairly heavyweight tool for a pattern of this complexity.

Reply:
These are not equivalent in the face of multiple newlines, but I'm not sure it matters. The chained replaces will allocate more memory, but this is not on a hot path.